src/app.tsx

```diff
@@ -85,12 +85,10 @@ const errorHandler = (error: { response: Response; data: any }): Promise<Respons
 
 const adminAPIConfig = getAdminAPIConfig();
 export const request: RequestConfig = {
+  prefix: `${adminAPIConfig.schema}://${adminAPIConfig.host}${adminAPIConfig.path}${adminAPIConfig.prefix}`,
```

Comment:
how about this?

```js
const {key, xxx, xxx} = getAdminAPIConfig()
```

Reply:
It is expected that this function will be cancelled in the later modification of the setting page.